en/asgardeo/docs/guides/users/user-stores/configure-a-user-store.md

```diff
@@ -1,32 +1,56 @@
 # Connect a remote user store
 
-!!! note
-    If you have connected a remote user store before 7th February 2023, you will have Read-Only access for the user store.
-    If you need Read-Write access for your user store on Asgardeo,
+Asgardeo allows you to connect a remote user store and onboard users and groups from it. The user store connection can either be with read-only or read/write permissions. Administrators can then start managing access to business applications in Asgardeo for these users and groups. This guide explains how you can set up a remote user store for your Asgardeo organization.
+
+## User store connections
+
+Asgardeo supports the following user store connections.
+
+### Optimized user store connection (recommended)
+
+Asgardeo has introduced **optimized user store connections**, designed for high scalability and performance. At the moment, it supports **read-only** user stores and can be used to authenticate users and retrieve attributes. This connection is continuously evolving and will support a wide range of use cases in the future.
+
+If this connection's capabilities meet your organization's requirements, we highly recommend that you use the optimized connection instead of the classic connection.
+
+### Classic user store connection
+
+The optimized user store connection is the recommended choice, though its capabilities are limited at the moment. If your organization's requirements exceed the capabilities offered by the optimized connection, you can use the **classic user store connection**. The classic connection supports both **read/write** and **read-only** user stores, and offers full management capabilities. While a classic connection offers more capabilities, it may not be as performant as the optimized connection.
```

Comment:
```suggestion
The optimized user store connection is the recommended choice, though its capabilities are limited at the moment. If your organization's requirements exceed the capabilities offered by the optimized connection, you can use the **classic user store connection**. The classic connection supports both **read/write** and **read-only** user stores, and offers full management capabilities. While the classic connection offers more capabilities, it may not be as performant as the optimized connection.
```